utils.go

```diff
@@ -0,0 +1,20 @@
+package check
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kr/pretty"
+)
+
+// formatUnequal will dump the actual and expected values into a textual
+// representation and return an error message containing a diff.
+func formatUnequal(obtained interface{}, expected interface{}) string {
+	diff := pretty.Diff(obtained, expected)
+	if len(diff) == 0 {
+		diff = []string{fmt.Sprintf("%p != %p", expected, obtained)}
```

Comment:
This looks suspect. It doesn't seem better than what we already do for that case.

What's the background?

Reply:
I removed this again, this was to make it more inline with what #99 was doing (which uses a different diff library). But I agree that this adds no information.